src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/SQLFunctionsIT.java

```diff
@@ -242,6 +236,93 @@ public void functionPow() throws Exception {
         );
     }
 
+    @Test
+    public void operatorSubstring() {
+        assertThat(
+                executeQuery("SELECT substring('sampleName', 1, 4) FROM " + TEST_INDEX_ACCOUNT,
+                        "jdbc"),
+                containsString("samp")
```

Comment:
To make sure we are actually getting a substring, include an assert here to check it does not contains `sampleName ` i.e. 

```
executeQuery("SELECT substring('sampleName', 1, 4) FROM " + TEST_INDEX_ACCOUNT, "jdbc"),
               not( containsString("sampleName"))
```

Reply:
Good point, will do it! Thanks! @abbashus